web/src/scripts/components/forms/FormHeader.jsx

```diff
@@ -17,50 +17,61 @@
 
 import React, { Component, } from 'react'
 
-const style = {
-  height: 30,
-  display: 'flex',
-  flexDirection: 'row',
-  alignItems: 'center',
-  paddingRight: 10,
-  paddingLeft: 10,
-  minWidth: 0,
+const styles = {
+  container: {
+    height: 30,
+    display: 'flex',
+    flexDirection: 'row',
+    alignItems: 'center',
+    paddingRight: 10,
+    paddingLeft: 10,
+    minWidth: 0,
+  },
+  button: {
+    fontSize: 18,
+    fontWeight: 300,
+  }
 }
 
-const FormHeader = ({label, inset, labelWidth, disabled}) => {
+export default class extends Component {
```

Comment:
I was using functional components for a while, but:
1. they are less performant, as they can't have a `shouldComponentUpdate`
2. they are inconsistent with the class format

I'm going to use all class components from now on for consistency.

This PR is a bit lazy since I'm short on time, and I picked up many changes from a previous branch. So there are a lot of mixed styles.